### ISSUE TITLE
Update Glowstone++

### ISSUE THREAD

**Issue by mastercoms** [EurekaMineTeam/EurekaMine-MP#1]
You should update the submodule for Glowstone++ for the latest fixes on our side ;)

**Comment by Edwardthedog2** [comment]
Okay, I have started working on it. Do all files need updated, or just specific ones?

**Comment by matthew4444** [comment]
Can u add support for pocketmien plugins?

**Comment by Edwardthedog2** [comment]
@matthew4444 That was one of Dragonet Team's original ideas, to add support for PHP plugins. However I took it out, because PHP is too confusing for me. I can't bring it back unless someone with good knowledge of PHP and Java  is willing to help.

**Comment by matthew4444** [comment]
Can u get hittmana and our plugin dev team to help?

**Comment by Edwardthedog2** [comment]
If @HittmanA and the rest of the team knows Java, as well as PHP, than I would love it for you guys to help. I know Java, but am not a good teacher, for teaching anything.

**Comment by matthew4444** [comment]
Kk thx ill ask

**Comment by HittmanA** [comment]
I don't know any Java. Maybe I can learn? Does Java work on Macs?

**Comment by matthew4444** [comment]
i think so

**Comment by Edwardthedog2** [comment]
Yes, it does indeed. Just download it from http://java.com. You can learn from http://www.learnjavaonline.org/. Also a lot of the code on this software is coming from Bukkit's source code, whitch you can browse at: https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse

**Comment by matthew4444** [comment]
can i also get help with php?

**Comment by matthew4444** [comment]
just saying

**Comment by Edwardthedog2** [comment]
@matthew4444 Contact me on Gitter, if you want to/need to get help.

**Comment by matthew4444** [comment]
how whcih gitter site,and i got a buddy who can make javascri[pt plugins compatile with this i think,and can we make a android version of this app and apk for download in android repo soon to be created plz?

**Comment by DefinitlyEvil** [comment]
It's nice to see that Dragonet is get going again. :D

**Comment by matthew4444** [comment]
and thx and wanna be a meber?

**Comment by matthew4444** [comment]
member?

**Comment by matthew4444** [comment]
i gotta install win7-10 on vmware for my mom shes still using windows vista lol

**Comment by matthew4444** [comment]
see u later

**Comment by Edwardthedog2** [comment]
@DefinitlyEvil Yeah, I'm doing it, because I've always wanted to develop an entire software, not just make plugins for others, but also because I loved Dragonet, and used it a lot. I just couldn't stand it being inactive. However, I am having a small problem with building the jar file with maven. Do you think you could help me with that please?

**Comment by DefinitlyEvil** [comment]
@Edwardthedog2 
Nice! And as for your problem, install Maven and add it to your `Path` system variable then try these: 
```bash
cd Glowkit
mvn clean install
cd ..
mvn clean package
```

**Comment by TheDiamondYT1** [comment]
Please add the following line in the README, if possible.

```EurekaMine-MP is a continuation of the [Dragonet Server Software](https://GitHub.com/DragonetMC/Dragonet). ```

**Comment by Edwardthedog2** [comment]
@TheDiamondYT1 Done

**Comment by TekkitCommando** [comment]
I am not sure what I would need to do but I know PHP.

**Comment by Edwardthedog2** [comment]
@TekkitCommando Do you know Java?
@DefinitlyEvil It still doesn't work.

**Comment by matthew4444** [comment]
Czn someone help me with this?https://github.com/Code-Adic-Team/v86

**Comment by TekkitCommando** [comment]
@Edwardthedog2 I do but haven't used it for anything big in a while, I am just getting back into the big stuff because of Byte Powered and Minecraft mod related stuff.

**Comment by matthew4444** [comment]
Recruiting members https://github.com/Code-Adic-Team/v86/issues/2
